Webshop.Catalog.Solution/Webshop.Order.API.Test/OrderAPITests.cs

```diff
@@ -58,7 +59,7 @@ public async Task GetAllOrders_ValidRequest_ReturnsIEnumerableOfOrderDto()
             var expectedData = new[] {
                 new {
                     Id = 1,
-                    Customer = "Pavol Hal�s",
+                    Customer = "Pavol Halás",
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion (non-blocking):** ‏It might be best if we don't use non-UTF-8 characters here, because it might end up having a stroke on other machines.

Reply:
Done